libenkf/include/ert/enkf/model_config.h

```diff
@@ -46,6 +46,7 @@ extern "C" {
   typedef struct model_config_struct model_config_type;
   const char *           model_config_get_data_root( const model_config_type * model_config );
   void                   model_config_set_data_root( model_config_type * model_config , const char * data_root);
+  bool                   model_config_data_root_set( const model_config_type * model_config );
```

Comment:
Could you rename this function as `model_config_data_root_is_set` to highlight the difference to model_config_set_data_root?

Reply:
Done